susemanager-utils/susemanager-sls/susemanager-sls.spec

```diff
@@ -33,7 +33,8 @@ Requires:       susemanager-build-keys-web >= 12.0.1
 BuildRequires:  python3-pytest
 BuildRequires:  python3-mock
 BuildRequires:  python3-salt
-Requires:       python3-PyYAML >= 5.1
+# Different package names for SUSE and RHEL:
+Requires:       (python3-PyYAML >= 5.1 or python3-pyyaml >= 5.1)
```

Comment:
Uhh :-)

I need to find out if this works already with openSUSE and current SLE version.
This also reminds me, that we need to implement these metadata parsing and re-generate in reposync and taskomatic.

Reply:
This works on Leap 15.2 (builds on copr and installs with the python-PyYAML dependency): https://copr.fedorainfracloud.org/coprs/sbluhm/uyuni-server/build/1775065/

Reply:
It is supported since SLE15 GA and it seems we have chances that such packages works also out of the box with uyuni. So let's give it a try.

Reply:
Interesting, I just had a different attempt failing on an OBS build. might have been a wrong syntax on that. Definitely worth monitoring.